gui/packages/ubuntupro/lib/pages/subscribe_now/subscribe_now_page.dart

```diff
@@ -26,83 +29,82 @@ class SubscribeNowPage extends StatelessWidget {
     final linkStyle = MarkdownStyleSheet.fromTheme(
       theme.copyWith(
         textTheme: theme.textTheme.copyWith(
-          bodyMedium: theme.textTheme.bodyLarge,
+          bodyMedium: theme.textTheme.bodyMedium,
         ),
       ),
     );
 
-    return LandingPage(
-      children: [
-        SizedBox(
-          width: 400,
-          child: MarkdownBody(
-            data:
-                lang.proHeading('[${lang.learnMore}](https://ubuntu.com/pro)'),
-            onTapLink: (_, href, __) => launchUrlString(href!),
-            styleSheet: linkStyle,
-          ),
+    return ColumnPage(
+      left: [
+        MarkdownBody(
+          data: lang.proHeading('[${lang.learnMore}](https://ubuntu.com/pro)'),
+          onTapLink: (_, href, __) => launchUrlString(href!),
+          styleSheet: linkStyle,
         ),
-        const SizedBox(height: 16),
-        Row(
-          mainAxisAlignment: MainAxisAlignment.start,
-          children: [
-            if (model.purchaseAllowed) ...[
-              ElevatedButton(
-                onPressed: model.purchaseAllowed
-                    ? () async {
-                        final subs = await model.purchaseSubscription();
+        if (model.purchaseAllowed) ...[
+          const SizedBox(height: 16.0),
+          OutlinedButton(
+            onPressed: model.purchaseAllowed
+                ? () async {
+                    final subs = await model.purchaseSubscription();
```

Comment:
Juan's design suggested to add a button that redirects to `u.c/pro/subscribe` if MS Store Purchase is not implemented/allowed. Did you notice that?

Reply:
I didn't initially, but I think that's a good idea so I've implemented it